_posts/2020/11/2020-11-06-configure-artipie-maven-repo.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+date: 2020-11-06
+title: "Using Artipie as a maven repository"
+author: Alexander Krasnov
+description: This blog post explains how to use Artipie as a Maven repository.
+keywords:
+  - artipie
+  - private
+  - maven
+  - repository
+  - registry
+  - java
+---
+
+## Introduction
+
+As we know, Apache Maven is one of the most popular build management tool for Java. Many companies that do Java development have private Maven repositories, where they store all the artifacts. There are some problems with management when many developers work with a huge repository.    
+Let's assume, you need to restrict access to artifacts. So, it's necessary to split this one into several small repositories or add authentication for users. But at the same time you want to keep the ability to share easily and quickly within a team or between multiple teams of developers. Or you are just wondering how to set up an own repository for a pet project.  
+[Artipie](https://github.com/artipie/artipie) allows you to solve these problems as it is a binary artifact management tool. In the blog post I will share with you how to configure Artipie and project for this purpose. Let's start!
+
+## Installation
+
+In the simplest case, you need a file with the contents below. We want to use file storage therefore we specify `fs` as the type. At first, create a separate directory with the following `artipie.yaml` file inside:
+
+```yml
+meta:
+  storage:
+    type: fs
+    path: /var/artipie/configs
+```
+
+After that you can install and run Artipie with docker:
```

Comment:
@genryxy what is "docker"? Maybe a cross link would be helpful? Also, it's Docker, not docker.